SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorSettings.h

```diff
@@ -111,21 +114,20 @@ struct FWorkerTypeLaunchSection
 	GENERATED_BODY()
 
 	FWorkerTypeLaunchSection()
-		: WorkerTypeName()
-		, WorkerPermissions()
+		: WorkerPermissions()
 		, MaxConnectionCapacityLimit(0)
 		, bLoginRateLimitEnabled(false)
 		, LoginRateLimit()
-		, Columns(1)
-		, Rows(1)
+		, bAutoNumEditorInstances(true)
 		, NumEditorInstances(1)
 		, bManualWorkerConnectionOnly(true)
+		, WorkerLoadBalancing(nullptr)
 	{
 	}
 
-	/** The name of the worker type, defined in the filename of its spatialos.<worker_type>.worker.json file. */
-	UPROPERTY(Category = "SpatialGDK", EditAnywhere, config)
-	FName WorkerTypeName;
+	/** Worker type name, deprecated in favor of defining them in the runtime settings.*/
+	UPROPERTY(config)
+	FName WorkerTypeName_DEPRECATED;
```

Comment:
Oh I get it, I do feel this might be confusing though. Have we thought about what happens if a user gets mixed up here?

Reply:
The _DEPRECATED properties are hidden. So users should never see them.